### PR TITLE
Project Etch A Sketch Updated

### PR DESCRIPTION
**Pull request by Phildeoner**
-Create a button to prompt user to input grid size to be created
-Create a function generateGrid to accept user input and create grid of equal rows and columns
-Update the function to clear grids and create a new grid when new input is made
-Grid should display color on hover and leave pixelated trails like a pen.

### CODE REVIEW

File: stylesheet.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
+*{
+    margin: 0;
+    padding: 0;
+}
+.gridBoxes:hover {
+    box-shadow: 0 0 0.5em black;
```
Comment (zizoh) [inline stylesheet.css]: Please what does the "em" in "0.5em" mean?

**Review by zizoh — APPROVED** [review]

**Review by DanSam5K — APPROVED** [review]
Hi @Phildeoner, ✌️Approved

- It would be great if you can be consistent with using same unit of on sizes.  Currently your css uses "px" and "em" which has raise question from earlier review. How about fixing it before merging? 😉 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

File: stylesheet.css (hunk 1 above)
Comment (Phildeoner) [reply stylesheet.css]: The em is a css measuring unit that equals the font size, but in this case it was an oversight i will change it cos it applies blur to the box.

**Comment by Phildeoner** [comment]
Thank you sir

File: stylesheet.css (hunk 1 above)
Comment (Phildeoner) [reply stylesheet.css]: It has been adjusted sir thanks

Comment (zizoh) [reply stylesheet.css]: Thank you for the explanation.

**Pull request merged**